src/qat/backend/transform_passes.py

```diff
@@ -79,3 +81,21 @@ def run(self, builder: InstructionBuilder, res_mgr: ResultManager, *args, **kwar
             unique_variables = set(acq.output_variable for acq in acquires)
 
         builder.returns(list(unique_variables))
+
+
+class DesugaringPass(TransformPass):
+    """
+    Transforms syntactic sugars and implicit effects. Not that it applies here but a classical
```

Comment:
```suggestion
    Transforms syntactic sugars and implicit effects. Note that it applies here but a classical
```

Reply:
nah it's a `Not` ^^, but fair point though 😄